airflow/contrib/operators/gcs_to_s3.py

```diff
@@ -101,7 +101,7 @@ def execute(self, context):
             # Google Cloud Storage and not in S3
             bucket_name, _ = S3Hook.parse_s3_url(self.dest_s3_key)
             existing_files = s3_hook.list_keys(bucket_name)
-            files = set(files) - set(existing_files)
+            files = list(set(files) - set(existing_files))
```

Comment:
not sure on this. `xcom` push should be able to support any pickleable object like list, set

Reply:
Actually the error log says it does not support `set`: ` TypeError: 'NoneType' object is not iterable.` but it works with list.

Reply:
added traceback on the description. According to the traceback model.py internally uses json.dumps(value).encode but it does not support set since set is not JSON serializable

Reply:
Yup, yup. You are right